### PR TITLE
Content permissions

### PR DESCRIPTION
**Pull request by fructusmortus**
A custom decorator to check user's permissions has been realized in currency's views, also added filter currency_extras.py to check the same in the all_currencies.html template

### CODE REVIEW

File: currency/views.py [hunk 1]
```diff
@@ -1,16 +1,33 @@
...
 
+def has_perm(self, perm, obj=None):
+    try:
+        user_perm = self.user_permissions.get(codename=perm)
+    except ObjectDoesNotExist:
+        user_perm = False
+    if user_perm:
+        return True
+    else:
+        return False
```
Comment (nargiza-sarkulova) [inline currency/views.py:13]: Should achieve the same:
```
def has_perm(self, perm, obj=None):
    try:
        user_perm = self.user_permissions.get(codename=perm)
    except ObjectDoesNotExist:
        return False
    if user_perm:
        return True
    return False
```

File: currency/templatetags/currency_extras.py [hunk 2]
```diff
@@ -0,0 +1,10 @@
+from django import template
+from django.contrib.auth.models import Group
+
+register = template.Library()
+
+
+@register.filter(name='is_group')
+def is_group(user, group_name):
```
Comment (nargiza-sarkulova) [inline currency/templatetags/currency_extras.py:8]: What `templatetags` mean? Why not create a file called `utils.py` in `currency` folder and keep this method there?

Comment (fructusmortus) [reply currency/templatetags/currency_extras.py:8]: It's a conventional way to realize some custom template tags
https://docs.djangoproject.com/en/3.2/howto/custom-template-tags/

File: currency/views.py (hunk 1 above)
Comment (fructusmortus) [reply currency/views.py:13]: You're right it has achieved.

**Review by nargiza-sarkulova — APPROVED** [review]

**Pull request merged**